packages/vue-app/template/utils.js

```diff
@@ -10,6 +10,15 @@ if (process.client) {
   }
 }
 
+export function createGetCounter (counterObject, defaultKey = '') {
+  return function getCounter (id = defaultKey) {
+    if (counterObject[id] === undefined) {
```

Comment:
Since we have this check, do we still need default `'': 0`?

Reply:
Good point :+1: